Qa 4 8

DeepakAtariya opened the pull request:
### Description

Please describe your pull request.

Fixes #

Before submitting pull request, please review the following checklist:

- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

saurass committed:
fixes #14896 - [QA_4_8] Expanded DB, refresh, plus/minus icon

Signed-off-by: saurass <saurabhsrivastava312@gmail.com>

saurass committed:
reduce elseif(s)

Signed-off-by: saurass <saurabhsrivastava312@gmail.com>

sijie123 committed:
ExportJson: add checks to prevent crash when encode returns boolean instead of string

Signed-off-by: Si Jie <sijie123@gmail.com>

kartik1000 committed:
Fix #14402 - Javascript error adding index to a new column

Fixes: #14402
Signed-off-by: Kartik Kathuria <kathuriakartik0@gmail.com>

wwycheuk committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (3297 of 3297 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/zh_Hant/
Signed-off-by: Walter Cheuk <wwycheuk@gmail.com>

weblate committed:
Merge branch 'origin/QA_4_8' into Weblate.

tmtisfree committed:
Translated using Weblate (French)

Currently translated at 100.0% (3298 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/fr/
Signed-off-by: tmtisfree <tmtisfree@free.fr>

Commit:
Translated using Weblate (Japanese)

Currently translated at 99.9% (3296 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/ja/
Signed-off-by: Joshua PowellNishiyama <nzjpnboy@outlook.com>

shucon commented:
Please explain your Pull Request and properly sign it off.
Refer to https://github.com/phpmyadmin/phpmyadmin/wiki/Developer_guidelines

Commit:
Translated using Weblate (Japanese)

Currently translated at 99.9% (3296 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/4.8
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/4-8/ja/
Signed-off-by: worldwideskier <worldwideskier@yahoo.co.jp>

williamdes committed:
Remove space in header X-Forwarded-For

Signed-off-by: William Desportes <williamdes@wdes.fr>

williamdes committed:
Add ChangeLog entry for #14402

[ci skip]

Signed-off-by: William Desportes <williamdes@wdes.fr>

williamdes committed:
Fix: #14896 - Expanded DB, refresh, plus/minus icon

Signed-off-by: William Desportes <williamdes@wdes.fr>

williamdes committed:
Add ChangeLog entry for #14896

[ci skip]
Signed-off-by: William Desportes <williamdes@wdes.fr>

williamdes committed:
Fix: #14922 - Json encode error

Fixes: #14922
Signed-off-by: William Desportes <williamdes@wdes.fr>

williamdes committed:
Add ChangeLog entry for #14922

[ci skip]

Signed-off-by: William Desportes <williamdes@wdes.fr>

ibennetch commented:
@DeepakAtariya I think something went wrong with creating your pull request, I think this just contains commits that are already part of QA_4_8. Perhaps you didn't commit your changes before submitting them?
